### ISSUE TITLE
[Feature][Metadata] Requirements for metadata gathering(元数据需求收集)

### ISSUE THREAD

**Issue by wanshicheng** [DataLinkDC/dinky#681]
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dlink/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Do you have a good idea for metadata? Please write it down.

Metadata includes technical metadata, business metadata, administrative metadata, and so on. The focus here is on technical metadata and administrative metadata.

你对元数据有什么好的想法，请在下面。

元数据包含技术元数、业务元数据、管理元数据等等。这里主要讨论技术元数据和管理元数据。

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by complone** [comment]
在没有元数据管理的情况下，从使用问题、管理问题、数据问题等三个方面进行说明：

### 使用问题

- 查看表结构信息不方便（查看、检索、表复用）
- 表依赖关系不清楚（血缘）
- 表信息缺少（表维护者，表状态）
### 管理问题

- 表权限管理
- 数据管理
- 数据质量监控
### 数据问题

- 建表规则混乱，没有统一标准
- 表结构变更之后，信息不同步

故而建议roadmap 如下:
![roadmap](https://user-images.githubusercontent.com/20021404/177236790-1493a9ad-bc61-4980-8624-143c7f36ae94.png)


流程:
![1112](https://user-images.githubusercontent.com/20021404/177236959-90952829-d80b-46f8-b4d9-24cba80ffae8.png)

以阿里云的oops举例:,数仓对于B端的交易报表统计流程如下:
![1113](https://user-images.githubusercontent.com/20021404/177237082-5e7fd522-39b6-4f26-aa36-5ae0750e4a67.png)

**Comment by aiwenmo** [comment]
目前数据源元数据已实现，计划在 0.7.0 的元数据中心的实现中，需要先实现一下功能：
1.逻辑表和字段的管理；
2.物理表元数据采集到逻辑表；
3.逻辑表元数据同步至物理表。
后续其他功能将在 0.7.0 版本中进行扩展。

**Comment by gitfortian** [comment]
@complone  good idea   目前元数据管理模块确实太简单了  基本就只是一个展示  按照阿里的这个设计 确实很合理  如果有接下来的roadmap  我这边可以参与贡献

**Comment by codegit10001** [comment]
挺好的功能，支持一下
